### PR TITLE
Added Bluedog Design Bureau parts to tech tree.

### PR DESCRIPTION
**Pull request by Jerre122**
Added BDB fasa clamps and towers, and BDB diamant rocket.
Added .vs directory to .gitignore

### CODE REVIEW

File: tree.yml [hunk 1]
```diff
@@ -1393,7 +1430,7 @@ basicConstruction:
     r7_blok_bvgd_sput: &R7Booster
         cost: 100
     r7_blok_a_sput:
-        cost: 120	
+        cost: 120
```
Comment (stratochief66) [inline tree.yml]: @NathanKell 
will trailing changes like this have any effect, or no?

**Comment by stratochief66** [comment]
Tentatively, looks good to me.

File: tree.yml (hunk 1 above)
Comment (NathanKell) [inline tree.yml]: I don't think so. Travis would complain if the tree didn't compile.

**Comment by pap1723** [comment]
I know this is an old PR, but with the Tree.yml being deprecated, and the old tree not being used anymore, this no longer applies. All BDB parts should be placed correctly in the new tree already.

**Pull request closed without merging**